lib/multiplex-mappers.cc

```diff
@@ -178,6 +178,80 @@ class CoremanMapper : public MultiplexMapperBase {
   }
 };
 
+class P10OutdoorMapper : public MultiplexMapperBase {
+public:
+  P10OutdoorMapper() : MultiplexMapperBase("p10outdoor", 2) {}
```

Comment:
Instead of P10OutdoorMapper (which is essentially all the mappers you find in this file), can you name it according to the characterisitc it does ? Above, there is for instance a mapper that 'snakes' through the pixels, thus its name. Is there anything you could describe what this mapper is doing to give it a nice distinctive name ?